pywbem/_mof_compiler.py

```diff
@@ -1689,7 +1689,6 @@ def p_scope(p):
               'PARAMETER',
               'ANY'):
         scopes[i] = i in slist
```

Comment:
How is an invalid scope detected? This parsing function just ignores invalid scopes, so I think the TODO was right on.

Reply:
Actually the code is correct.  The test is done in metaElement.  However, metaElement is wrong in that it includes elements that are not valid scopes (i.e. QUALIFIER).  Note that p_scopes() is the only use of metaElementList and metaElement.  That one confused me for a while.